lib/controllers/ActuatorController.cpp

```diff
@@ -217,4 +217,84 @@ bool ActuatorController::isPastMinAngle() {
 
 bool ActuatorController::isPastMaxAngle() {
   return (getAngle_Degrees() > m_actuatorConfig.maxAngle_Degrees || isLimSwitchMaxTriggered());
-}
\ No newline at end of file
+}
+
+// Update m_actuatorConfig upon reception of new PID values
+void ActuatorController::updateActuatorConfigP(const float& newP, const bool& velocityPID) {
```

Comment:
just being picky...but could you rename velocityPID to sth more like a boolean, eg, isVelocityPID?

Reply:
sure thing